int32 id
time src_time
string src_frame
float64 confidence

MarkerGroup[] marker_groups
OrientationGroup[] orientation_groups
